<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administratorHomePage</class>
 <widget class="QWidget" name="administratorHomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>点单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>修改菜单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>员工休假审批</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>查看薪水</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_11">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>管理食材数量</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>查询销售额</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>管理顾客评论</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>修改密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>管理桌位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>打卡上下班</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>返回</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
